docs/openshift/kctlr-use-as3-backend.rst

```diff
@@ -13,7 +13,7 @@ Prerequisites
 
 Ensure you meet the following requirements:
 
-- BIG-IP Controller v1.10 or higher when used in OpenShift.
+- BIG-IP Controller v1.11 or higher when used in OpenShift.
```

Comment:
```suggestion
- BIG-IP Controller v1.10 or higher when used in OpenShift.
```
This feature is introduced in 1.10. Minimum requirements should be kept at that version.

Reply:
Ok Sure